### ISSUE TITLE
Breakpoints do not work properly, again, and still.

### ISSUE THREAD

**Issue by stratboy** [kenwheeler/slick#3345]
I know you'll do exactly nothing, because it's what you did since the first tipe you got this kind of request. But I'll write it again, because it's the right thing to do. Tipical use case:

``` js
 $(".slider").slick({
    mobileFirst: true,

    responsive: [{
      breakpoint: 830,
      settings: "unslick"
    }]
}
```

If you start with a small screen, it wil work. As soon as you reach 830 it will unslick. Fine. Than go back to < 830, and you won't have slick back.

So you cannot say you're plugin is responsive. It's not. A plugin called responsive should behave like a responsive thing. And this is not the case. I think after 2+ years. I come back and guess? Still the same issue.

**Comment by eballeste** [comment]
listen for the window resize event and reinitialize slick
```
off = windowWidth <= 830;

onResize:
  if (windowWidth <= 830 && !off) {
    unslick
    off = true
  } else if (windowWidth > 830) {
    off = false
  }

**Comment by dietantejensen** [comment]
reinitializing is the logical thing to do - but I wonder how many people worried and wondered what they did wrong; especially, when the slider works like a charm most of the time.

**Comment by eparisio** [comment]
it should be implemented
same case here, it should be more clean and fast having an internal method instead of custom external function i think

**Comment by stratboy** [comment]
@eparisio it has been at least 3 years since my very first request about this...

**Comment by eballeste** [comment]
this repo has not been updated since april. 2018. i think it's time for everybody to move on from slick slider.

**Comment by mscoobby** [comment]
@stratboy The feature you request is more like "disable" rather than "destroy". And the whole point of destruction is to clean up itself so once destroyed they can't initialised it again. Here a "disable" feature can be implemented but to be honest simple event listener on the resize event does the job perfectly as @eballeste mentioned in his comment.

**Comment by stratboy** [comment]
@mscoobby I really think that a 'responsive' functionality must be bi-directional.

**Comment by mscoobby** [comment]
> @mscoobby I really think that a 'responsive' functionality must be bi-directional.

@stratboy Yes I agree. They should implement a function to disable it on certain breakpoints, but the destroy function is also useful in some cases.

**Comment by mczernow** [comment]
When **unslick** happens, this line is being triggered:
https://github.com/kenwheeler/slick/blob/master/slick/slick.js#L803

because of that **resize** method won't be triggered anymore, ever. And this is the very method that would trigger evaluating breakpoints. It seems that fixing this won't be that hard, because if we allow to evaluate breakpoints, slick will trigger **refresh**:
https://github.com/kenwheeler/slick/blob/master/slick/slick.js#L644
if for given breakpoint, the **settings** property are not **unslick**.
**refresh** will then lead to **init** method, and it's imaginable that we should see the slick to be ready again.

Of course, it may not be that easy. Hard to tell how the system will behave, if we allow it to be active all the time. But it seems to considerably easy to fix it.

**Comment by ejazrasool** [comment]
> > @mscoobby I really think that a 'responsive' functionality must be bi-directional.
> 
> @stratboy Yes I agree. They should implement a function to disable it on certain breakpoints, but the destroy function is also useful in some cases.

That is true as well, but disable vs. destroy is an implementation detail and the resulting behavior doesn't reflect the literal first two lines of slick website (at the time of writing)

> Fully responsive. Scales with its container.
Separate settings per breakpoint

_Separate settings per breakpoint_, nah, in reality one breakpoint (one having `settings: "unslick"` supersedes all others.

As you said, time to move on.

**Comment by eballeste** [comment]
move on people! the issues are about to reach 1000 open issues and apparently the only merged pull requests are from people who talk to directly to the main dev. stop beating a dead horse.

**Comment by abhishek98s** [comment]
### Here is the solution
https://github.com/kenwheeler/slick/issues/3324#issuecomment-2674463014
